src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
```

Comment:
Ensure that the image `../images/star.svg` exists at the specified path. If the image is missing, the stars will not display correctly.